ansible/roles/compiled/tasks/astro/calceph.yaml

```diff
@@ -1,4 +1,9 @@
 - name: Install calceph
+  environment:
+    MAKEFLAGS: "-j {{  ansible_processor_vcpus }}"
+    FFLAGS: "-fallow-argument-mismatch"
+    CC: "gcc-10"
+    FC: "gfortran-10"
```

Comment:
Was thinking that `environment` can be done on block level, and it can. Quite a few plays can be simplified this way…

Reply:
I saw the environment stuff in the other PR's. But yeah this could make it way easier to specify!